microsoft/vscode-azureresourcegroups
# Apply cloud shell bug fixes

Bug fixes that will be applied on top of the initial cloud shell migration #848. Putting this PR up for testing purposes. I'll explain each fix individually later.

How to test:

Make sure to run webpack before testing locally. The feature relies on the `cloudShellLauncher.ts` file being webpacked and put in `dist`.

<img width="327" alt="image" src="https://github.com/microsoft/vscode-azureresourcegroups/assets/12476526/c2b57d22-d086-48c2-8417-be99280ec549">
